ting.entities.inc

```diff
@@ -316,6 +317,10 @@ class TingEntity extends DingEntity {
       drupal_alter('ting_online_url', $url, $this);
     }
 
+    if(strpos($url, '[useraccessinfomedia]') === 0){
+      drupal_alter('infomedia_url', $url, $this);
+    }
```

Comment:
I do not think the right way to go about this is to introduce a new hook when we already have `ting_online_url`.

As I see it the problem is that infomedia urls are not identified as proper  urls in line 303 as they do not begin with 'http'. In that case I would much prefer that the detection code is updated to support urls with a leading token instead of one-off solutions.